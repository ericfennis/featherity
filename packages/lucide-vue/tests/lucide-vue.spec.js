import { mount } from '@vue/test-utils'
import { Smile } from '..'

describe('Using lucide icon components', () => {
  it('should render an component', () => {
    const wrapper = mount(Smile)
    expect(wrapper).toMatchSnapshot();
  });

  it('should adjust the size, stroke color and stroke width', () => {
    const wrapper = mount(Smile, {
      propsData: {
        size: 48,
        stroke: 'red',
        strokeWidth: 4
      }
    })

    expect(wrapper).toMatchSnapshot();
  });
});
