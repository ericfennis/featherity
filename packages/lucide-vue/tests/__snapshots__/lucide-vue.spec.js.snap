// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Using lucide icon components should adjust the size, stroke color and stroke width 1`] = `
<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 24 24" fill="none" stroke="red" stroke-width="4" stroke-linecap="round" stroke-linejoin="round">
  <circle cx="12" cy="12" r="10" key="14id60"></circle>
  <path d="M8 14s1.5 2 4 2 4-2 4-2" key="1qsru9"></path>
  <line x1="9" y1="9" x2="9.01" y2="9" key="sl5tun"></line>
  <line x1="15" y1="9" x2="15.01" y2="9" key="136pav"></line>
</svg>
`;

exports[`Using lucide icon components should render an component 1`] = `
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
  <circle cx="12" cy="12" r="10" key="14id60"></circle>
  <path d="M8 14s1.5 2 4 2 4-2 4-2" key="1qsru9"></path>
  <line x1="9" y1="9" x2="9.01" y2="9" key="sl5tun"></line>
  <line x1="15" y1="9" x2="15.01" y2="9" key="136pav"></line>
</svg>
`;
